Xamarin.Forms.Platform.iOS/Renderers/ShellTableViewController.cs

```diff
@@ -91,5 +102,26 @@ void OnScrolled(object sender, UIScrollView e)
 
 			LayoutParallax();
 		}
+
+		float SafeAreaOffset
+		{
+			get
+			{
+				if (!Forms.IsiOS11OrNewer)
```

Comment:
maybe better to save the first value in a variable then give it?

Reply:
the problem with saving it is that the safe area changes on device rotation so we could detect for that then clear it out but this seems safer. 

The way this all works with a rotated device + notch + rounded corners needs to be fleshed out a bit